src/components/Table/Table.stories.mdx

```diff
@@ -428,11 +438,12 @@ Setting the `isLoading` prop to true will disable clicking on the column headers
         { id: 3, color: 'green', flavor: '' },
       ];
       return (
-        <Table
-          columns={columnConfig}
-          tableData={tableData}
-          emptyCellPlaceholder="--"
-        />
+        <Table isStriped>
```

Comment:
Let's remove `isStriped` for this example to clarify that we are only demonstrating the placeholder.